etc/releasing.md

```diff
@@ -450,6 +452,28 @@ $ DEBEMAIL='my-email@mongodb.com' DEBFULLNAME='FIRSTNAME LASTNAME' dch -r -D exp
   especially the changelog on the master remain up to date)
 - Open a PR with the cherry-picks and use the `Rebase and merge` merge strategy.
 
+#### Post Build
+
+After the package has been accepted into Debian, make sure to check if there are
+any bugs or build failures. Debian tests builds on more architectures than we
+officially support and test on in Evergreen, so there are occasionally
+surprising build failures on exotic architectures.
+
+Bug tracking can be found here for the following projects:
+- [libmongocrypt](https://bugs.debian.org/cgi-bin/pkgreport.cgi?src=libmongocrypt)
+- [mongo-c-driver](https://bugs.debian.org/cgi-bin/pkgreport.cgi?src=mongo-c-driver)
+- [mongo-cxx-driver](https://bugs.debian.org/cgi-bin/pkgreport.cgi?src=mongo-cxx-driver)
+
```

Comment:
```suggestion
Any Debian bugs that were specifically fixed in the release being made should be noted in the `debian/changelog` entry. It should look something like this:

    `* New upstream release (Closes: #1234567)`
```

Reply:
I like it! I'll find a way to add it in. Thanks!